scripts/build.ts

```diff
@@ -413,6 +413,11 @@ export default async function (
             }
           } else if ((obj[depName] as string).match(/\b0\.0\.0-PLACEHOLDER\b/)) {
             obj[depName] = (obj[depName] as string).replace(/\b0\.0\.0-PLACEHOLDER\b/, v);
+          } else if ((obj[depName] as string).match(/\b0\.0\.0-EXPERIMENTAL-PLACEHOLDER\b/)) {
+            obj[depName] = (obj[depName] as string).replace(
+              /\b0\.0\.0-EXPERIMENTAL-PLACEHOLDER\b/,
+              v,
+            );
```

Comment:
Not ideal to add this extra condition here but this build script will be deleted after flipping to the bazel build script.